pom.xml

```diff
@@ -0,0 +1,215 @@
+                    <rulesets>
+                        <ruleset>src/main/resources/plugins/pmd-ruleset.xml</ruleset>
+                    </rulesets>
+                    <printFailingErrors>true</printFailingErrors>
+                    <showPmdLog>true</showPmdLog>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>3.2.0</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/plugins/*</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+    <reporting>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-pmd-plugin</artifactId>
+                <version>3.12.0</version>
+                <configuration>
+                    <rulesets>
+                        <ruleset>src/main/resources/plugins/pmd-ruleset.xml</ruleset>
+                    </rulesets>
+                </configuration>
+            </plugin>
+        </plugins>
+    </reporting>
+    <packaging>jar</packaging>
+    <dependencies>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-dal</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-dal-4.9.0.00409.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.dal</groupId>
+            <artifactId>symphony-dal-commons</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-dal-commons-4.9.0.00409.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>com.avispl.symphony.api</groupId>
+            <artifactId>symphony-api-commons</artifactId>
+            <version>${symphonyApiVersion}</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/symphony-api-commons-4.9.0.00409.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>org.icmp4j</groupId>
+            <artifactId>icmp4j</artifactId>
+            <version>1018</version>
+            <scope>system</scope>
+            <systemPath>${basedir}/lib/icmp4j-1018.jar</systemPath>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.sshd</groupId>
+            <artifactId>sshd-core</artifactId>
+            <version>2.2.0</version>
```

Comment:
What is that dependency for? Testing? Than it should have <scope>test</scope>. And please separate all testing dependencies in its own section. This applies to few other dependencies at the end of the list